packages/shared/src/components/modals/ContentWriteTab.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { ReactElement } from 'react';
+import { Modal } from './common/Modal';
+import CommentBox, { CommentBoxProps } from './CommentBox';
+import { Justify } from '../utilities';
+import { Button, ButtonSize } from '../buttons/Button';
+import AtIcon from '../icons/At';
+import { ClickableText } from '../buttons/ClickableText';
+import { markdownGuide } from '../../lib/constants';
+import { useUserMention } from '../../hooks/useUserMention';
+import { Post } from '../../graphql/posts';
+
+interface ContentWriteTabProps
+  extends Omit<CommentBoxProps, 'useUserMentionOptions'> {
+  post: Post;
+  tabName: string;
+}
+
+function ContentWriteTab({
```

Comment:
Fine for now, but in v6 we might consider doing sub folders, let me know what you guys think. Just more structure only.

Reply:
Agree with that. We will need to be more organized, I will create a `tabs` folder now in this `modals` folder.